_sass/button.scss

```diff
@@ -52,8 +54,11 @@ button {
         @include link-button;
         overflow: visible;
         border: 0;
-        color: $contentColor;
-        background: $fillColor;
+        color: $black;
+        background: $white;
+		border: 1px solid #dedede;
+  border-radius: 2px;
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
```

Comment:
let's define the box shadow in constant file, so we can use & change it everywhere.
https://github.com/ChampionFX/binary-style/blob/gh-pages/_sass/constants.scss